spec/rubocop/cop/rspec/factory_bot/attribute_defined_statically_spec.rb

```diff
@@ -86,6 +86,16 @@
     RUBY
   end
 
+  it 'accepts valid association definitions' do
+    expect_no_offenses(<<-RUBY)
+      FactoryBot.define do
+        factory :post do
+          author age: 42, factory: :user
```

Comment:
Do we need a test case for `association :author, age: 42, factory: :user`, or is that covered by our other code?

Reply:
I think that is in the same category as the sequence test below, so probably already covered well enough.